deploy/README.md

```diff
@@ -362,24 +364,81 @@ spec:
+  
+```yaml
+data:
+...
+my-custom-metrics: <base64EncodedURL>
+kind: Secret
+```
+
+  * Next you need to edit the FluentD Deployment and add a new environment variable, pointing to the new secret.  Assuming you installed the collection in the  `sumologic` namespace, you can run `kubectl -n sumologic edit deployment fluentd` or edit the YAML you deployed when you set up collection. Note, if you installed using helm, the name of the deployment may be different depending on how you installed the helm chart.
+  * Locate the `SUMO_ENDPOINT_LOGS` environment variable in the YAML and add a new environment variable that points to the secret key you created. The following is an example.
+  
+```yaml
+...
+        - name: SUMO_ENDPOINT_LOGS
+          valueFrom:
+            secretKeyRef:
+              key: endpoint-logs
+              name: sumologic
+        - name: MY_CUSTOM_METRICS
+          valueFrom:
+            secretKeyRef:
+              key: my-custom-metrics
+              name: sumologic
+        - name: LOG_FORMAT
+          value: fields
+```
+
+  * Finally, you need yo modify the fluentD config to route data to your newly created HTTP source. Assuming you installed the collection in the  `sumologic` namespace, you can run `kubectl -n sumologic edit configmap fluentd` or edit the YAML you deployed when you set up collection. Note, if you installed using helm, the name of the deployment may be different depending on how you installed the helm chart.
+  * Locate the section `match prometheus.metrics` and you will insert a new section above this.  The `match` statement should end with a tag that identifies your data that fluentD will use for routing.  Then make sure you point to the environment variable you added to your deployment. The following is an example.
+  
+```yaml
+...        
+          <match prometheus.metrics.YOUR_TAG>
+             @type sumologic
+             @id sumologic.endpoint.metrics
+             endpoint "#{ENV['MY_CUSTOM_METRICS']}"
+             @include metrics.output.conf
+           </match>
+          <match prometheus.metrics**>
+             @type sumologic
+             @id sumologic.endpoint.metrics
+             endpoint "#{ENV['SUMO_ENDPOINT_METRICS']}"
+             @include metrics.output.conf
+           </match>
+```
+
+#### Step 5: Update the prometheus-overrides.yaml file to forward the metrics to FluentD.
+
+The `prometheus-overrides.yaml` file controls what metrics get forwarded on to Sumo Logic. In order to get your custom metrics sending into Sumo Logic, you need to update the `prometheus-overrides.yaml` file to include a rule to forward on your custom metrics. Make sure you include the same tag you created in your FluentD configmap in the previous step. Here is an example addition to the `prometheus-overrides.yaml` that will forward metrics to Sumo:
```

Comment:
Change "In order to get your custom metrics sending into Sumo Logic, you need to update"
to
"To send custom metrics to Sumo Logic you need to update..."